modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/extensions/dataprovider/model/ActionResponse.java

```diff
@@ -16,7 +16,7 @@
  * under the License.
  */
 
-package org.wso2.identity.integration.test.actions.dataprovider.model;
+package org.wso2.identity.integration.test.extensions.dataprovider.model;
```

Comment:
let's rename as service-extensions

Reply:
Hyphens `-` are not allowed for package names. Hence shall we go with `extensions.service` ?

Reply:
Fixed with 87b85841aa35b32899b9a2f2871de7b4c0fa8d2b